tests/Calculator/CalculatorTest.php

```diff
@@ -84,6 +84,10 @@ public function testRound()
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_EVEN));
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_ODD));
         $this->assertEquals('12', $calculator->round('12.50', Money::ROUND_HALF_DOWN));
+        $this->assertEquals('-2', $calculator->round(-1.5, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.578947368, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.5, Money::ROUND_HALF_UP));
```

Comment:
Ah, no. No more responsibilities. The reason why people CAN describe rounding is to avoid problems originating from default behaviors.

All in all: I don't think there is a right solution for this problem. If we change, there will be problems. But this way it is misleading. Again: docs FTW, but apart from that, we can't really make everyone happy.